chart/fluentbit-operator/crds/logging.kubesphere.io_filter.yaml

```diff
@@ -395,6 +397,29 @@ spec:
                             type: string
                           type: array
                       type: object
+                    throttle:
+                      description: Throttle defines a Throttle configuration.
+                      properties:
+                        interval:
+                          description: Interval is the time interval expressed in
+                            "sleep" format. e.g. 3s, 1.5m, 0.5h, etc.
+                          pattern: ^\d+(\.[0-9]({0,2})?(s|m|h|d)?$
```

Comment:
There is an extra (  .
^\d+(\.[0-9]{0,2})?(s|m|h|d)?$

Reply:
:+1: Good find; Not sure how this snuck in 😮‍💨  ; all of the other CRD references have the correct pattern: https://github.com/fluent/fluentbit-operator/pull/120